Recommendation.java

```diff
@@ -23,6 +23,12 @@ public Recommendation()
       imgLink = "https://fl-1.cdn.flockler.com/embed/no-image.svg"
 
```

Comment:
Add a default name for the author instance variable inside of the Recommendation() constructor.

Reply:
THIS is the correct place to put author = "John Doe";